frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -55,6 +61,11 @@ export default {
     title() {
       return this.scheduleEntry.activity().title
     },
+    progressLabel() {
+      return this.scheduleEntry.activity().progressLabel !== null
+        ? this.scheduleEntry.activity().progressLabel().title
+        : null
+    },
```

Comment:
as i understand hal-json-vuex, the function should never be null... except when the activity is null.
return this.scheduleEntry.activity()?.progressLabel()?.title

Reply:
hm... but it is

![image](https://user-images.githubusercontent.com/470237/234110105-5e58c101-6682-42c5-9dbf-b244dc3c87f9.png)

Reply:
.... ok, you are right